### ISSUE TITLE
Left Nav - 1.3.1 Info and Relationships - 'Home' link is interpreted incorrectly as a list

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9711]
As a user using the keyboard and screen reader to navigate through the left-hand nav, the screen reader interpretation of the Home link should match what is visually presented on the page.

**Linked work item:**
#9740 

**Expected Result:**
As it is a link in and of itself and only part of a list of two items (Home and Search are the only links in the left hand nav)

**Actual Result:**
The Home link is interpreted as a list with four items - there are neither four items in the left hand nav, nor are there four items attached as a sublist to the Home link.

**Evidence:**
(Use NVDA to replicate)

**Comment by apeters** [comment]
@imogenroberts the title and description of the issue doesn't match.  Can you take a look and fix?

**Comment by johnatawnclementawn** [comment]
I think this will be addressed by #9740. Though as those changes stand, the hierarchy that is present in the left nav may not be intuitive for a user

**Comment by imogenroberts** [comment]
Hi Johnathan,

Just updated it to show the change now 😊

Thanks,
Imogen


[Historic England Logo]<http://www.historicengland.org.uk/>


Work with us to champion heritage and improve lives. Read our Future Strategy and get involved at historicengland.org.uk/strategy<http://historicengland.org.uk/strategy>.
Follow us:  Facebook<https://www.facebook.com/HistoricEngland>  |  Twitter<https://twitter.com/HistoricEngland>  |  Instagram<https://www.instagram.com/historicengland/>     Sign up to our newsletter<http://webmail.historicenglandservices.org.uk/k/Historic-England/historic_england_preference_centre>


This e-mail (and any attachments) is confidential and may contain personal views which are not the views of Historic England unless specifically stated. If you have received it in error, please delete it from your system and notify the sender immediately. Do not use, copy or disclose the information in any way nor act in reliance on it. Any information sent to Historic England may become publicly available. We respect your privacy and the use of your information. Please read our full privacy policy<https://www.historicengland.org.uk/terms/privacy-cookies/> for more information.

**Comment by johnatawnclementawn** [comment]
@imogenroberts This was the 'surprise' ticket I held off on talking about yesterday. Now that my improvements from #9740 have been merged into our Accessibility branch, I'm curious if this still fails.

**Comment by apeters** [comment]
@imogenroberts can you check this again?  I think it might be fixed in the latest.

**Comment by imogenroberts** [comment]
@apeters @johnatawnclementawn not yet fixed, but now only says a list with three items rather than four.

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest, can you recheck?

**Comment by imogenroberts** [comment]
RETEST: Home link is interpreted as a list with six items.  It is unclear from the subsequent navigation links which items are included in this list. 

**Evidence:** Use NVDA to replicate.

**Comment by imogenroberts** [comment]
RETEST: Home link is interpreted correctly. 

**Evidence:**


https://github.com/archesproject/arches/assets/77161085/7e78afc5-a45d-48a8-ac9e-c53fd978b265
